microsoft-edge/progressive-web-apps-chromium/how-to/icon-theme-color.md

```diff
@@ -60,10 +60,224 @@ Each icon in the `icons` array should come with at least the `src` and `sizes` p
+    * 44x44 - target size for taskbar, start menu, task manager
+    * 48x48 - target size for taskbar, start menu, task manager
+    * 60x60 - target size for taskbar, start menu, task manager
+    * 64x64 - target size for taskbar, start menu, task manager
+    * 72x72 - target size for taskbar, start menu, task manager
+    * 80x80 - target size for taskbar, start menu, task manager
+    * 96x96 - target size for taskbar, start menu, task manager
+    * 256x256 - target size for taskbar, start menu, task manager
+In level 4, you supply images for tiles with display scales and target size images for display in various surfaces in Windows, including taskbar, start menu, task manager, ALT+Tab task switcher, and more.
 
 <!-- ====================================================================== -->
+### Image level summary
+
+Consider, for example, how your app icon might show different content depending on the available surface area:
+
+* Small surface: App icon (44x44) in taskbar, showing a small, abbreviated logo: !
+[The small size icon](../media/windows-image-app-icon.png) 
+
+* Larger surface: Medium tile (150x150) in start menu, showing the full logo of the app:  
+![The medium size icon.](../media/windows-image-small-tile.png) 
+
+* Wide surface: Wide tile in start menu, showing a more articulated app icon: 
+![The wide size icon](../media/windows-image-wide-tile.png) 
+
+Notice how the content of the icon changed. This is possible only through the use of higher levels of icon support. We recommend developers choose higher levels to give the best experience for your Windows users.  
+
+<!-- ====================================================================== -->
+### Icon descriptions
+
+Below you’ll find a description of each app icon and where they show up in Windows.
+
+The normal app icon that's shown in the start menu, task bar, or task manager.
+
+![App Icon](../media/windows-image-app-icon.png) 
+
+* 44x44
+* 55x55 (1.25x scale)
+* 66x66 (1.5x scale)
+* 88x88 (2x scale)
+* 176x176 (4x scale)
+
+The small tile is shown in the start menu when the user set your app’s tile to small size.
+
+![Small tile](../media/windows-image-small-tile.png) 
+
+* 71x71
+* 89x89 (1.25x scale)
+* 107x107 (1.5x scale)
+* 142x142 (2x scale)
+* 284x284 (4x scale)
+
+The medium tile icon is shown in the start menu when the user sets your app’s tile to medium size.
+
+![Medium tile](../media/windows-image-medium-tile.png)
+
+* 150x150
+* 188x188 (1.25x scale)
+* 225x225 (1.5x scale)
+* 300x300 (2x scale)
+* 600x600 (4x scale)
+
+The wide tile icon is shown in the start menu when the user sets your app’s tile to wide size.
+
+![Wide tile](../media/windows-image-wide-tile.png)
+
+* 310x150
+* 388x188 (1.25x scale)
+* 465x225 (1.5x scale)
+* 620x300 (2x scale)
+* 1240x600 (4x scale)
+
+The large tile icon is in the start menu when the user sets your app’s tile to large size.
+![Large tile](../media/windows-image-large-tile.png)
```

Comment:
Missing empty line before this image. Please just add one empty line between "The large title icons..." and the "![..." lines.